transferui/src/main/java/com/hyperwallet/android/ui/transfer/viewmodel/CreateTransferViewModel.java

```diff
@@ -121,7 +121,14 @@
         mIsLoading.postValue(Boolean.TRUE);
         mIsCreateQuoteLoading.setValue(Boolean.FALSE);
         mShowFxRateChange.setValue(Boolean.FALSE);
-        loadTransferSource();
+    }
+
+    public void init() {
+        if (mSourceToken == null) {
```

Comment:
checking `mSourceToken` is not enough,
1st call `onActivityCreated()` no source token -> init() -> loadTransferSource() *it will now have `mSourceToken`
Succeeding call like rotate the phone `onActivityCreated()` source token from old init -> init() -> loadTransferSource() *it will now have `mSourceToken` 
*The bug will be visible when you select your your own transfer destination and rotate the phone, it will reinitialize again with the default source token from the 1st call

Reply:
Done